packages/testlab/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.log
+*.tgz
+package
+src
+test
+tsconfig.json
```

Comment:
I think we should be using an explicit white-list of files to include in the package via package.json `files` property, rather than maintaining a backlist in `.npmignore`.